citations/citations-impl/impl/src/java/org/sakaiproject/citation/impl/BaseCitationService.java

```diff
@@ -826,7 +828,22 @@ public void exportRis(StringBuilder buffer) throws IOException
 
 			// TODO: deal with real dates. Right now, just year
 
-  		exportRisField("Y1", getCitationProperty(Schema.YEAR, false) + "//", buffer);
+			exportRisField("Y1", getCitationProperty(Schema.YEAR, false) + "//", buffer);
+
+			// Extract the LCN from the otherIds citation field (if populated) and put in RIS U3.
+			String property = (String) getCitationProperty("otherIds", true);
+			if (property != null || property.length() > 0) {
```

Comment:
I would extract this out into method, then it makes it simple to unit test.

Reply:
Have done so.